index.html

```diff
@@ -6,47 +6,174 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
     <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
-    </head>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+</head>
 <body>
     <header>
-    <div class="pure-nav-list">
-            <h4 class="important"><a href="#">Ayodele</a></h4>
-            <nav>
-                <div class="menu1">
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                </div>
-            </nav>
-            </div>
-            
-        </header>
-
-        <main>
-    <section class="section-headline">
-    <div class="main-text Developer">
-    <div class="introduction">
-    <h1>Hey There. <br> I’m Ayodele</h1>
-    <h3>I’am a Software Developer</h3>
-    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-    
+    <a href="#"> Ayodele </a>
+    <nav class="toggle">
+        <ul>
+        <li><i class="fas fa-bars"></i></li>
+        </ul>
+    </nav>
+    </header>
+    <section class="content">
+        <h1>Hey There. <br> I’m Ayodele</h1>
+        <h3>I’am a Software Developer</h3>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <div class="item"><img src="img_social.png" alt="icons"></div>
+    <div id="background"></div>
+    </section>
+
+    <section class="picture">
+        <div class="work-section">
+            <h4>My Recent Works</h4>
```

Comment:
- [ ] You have done a good job with the showcase section :+1: but there is an issue with this title because it's aligned to the left and it should be centered according to the Figma design. Please, fix this alignment issue to match the reference :raised_hands:
- [ ] Also, the separator line below the section title is taking the entire viewport width and there should be spacing to the sides. Please, fix this issue to make your portfolio look even better :smiley:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144652067-ad1e8240-b84d-492c-acba-2104a717ae65.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144652125-b7f798ee-cd6f-475a-bc72-323f55cede26.png)

- [ ] According to the Figma design the card image should have the same width as the description and the button. I would suggest addressing this issue to match the reference :smiley: :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144652626-d66f2756-7523-43bd-b86a-0494cfd12d14.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144652662-891938b5-34d6-4564-8d4a-f98e9b9a42c3.png)